src/wrapper/jnienv.rs

```diff
@@ -1068,15 +1068,17 @@ impl<'a> JNIEnv<'a> {
         non_null!(array, "convert_byte_array array argument");
         let length = jni_non_void_call!(self.internal, GetArrayLength, array);
         let mut vec = vec![0u8; length as usize];
-        jni_void_call!(
-            self.internal,
-            GetByteArrayRegion,
-            array,
-            0,
-            length,
-            vec.as_mut_ptr() as *mut i8
-        );
-
+        unsafe {
+            jni_unchecked!(
+                self.internal,
+                GetByteArrayRegion,
+                array,
+                0,
+                length,
+                vec.as_mut_ptr() as *mut i8
+            );
+        }
+        check_exception!(self.internal);
```

Comment:
> I believe jni_unchecked is OK here because this method determines the start and end indices, therefore, ArrayIndexOutOfBoundsException is not possible.

Why adding a check for an impossible exception?

Reply:
Ah, you mean this. Ok, it was introduced more than a year ago within #15 and, most probably, this check is redundant. Going to remove it.